src/rest-server/src/utils/manager/user/user.js

```diff
@@ -33,13 +33,13 @@ const userSchema = Joi.object().keys({
   password: Joi.string()
     .empty('')
     .default(''),
-  extension: Joi.object().pattern(/\w+/, Joi.required()).required(),
+  extension: Joi.object().pattern(/\w+/, Joi.required()).default(),
```

Comment:
This filed must be a ```{}```, if you think replace ```required``` with default is better. Please set default value as ```{}``` (empty dict). This schema constraint the final structure in secret, pls check the data strictly here. If you wanna set any default value logic, you'd better put them in the controller or model level.

Reply:
Same reason as above